grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_query_helper/osquery_helper_interfaces.ts

```diff
@@ -0,0 +1,96 @@
+import {OsqueryTableSpec, nameToTable, OsqueryColumnSpec} from './osquery_table_specs';
```

Comment:
`osquery_helper_interface.ts` sounds too generic (you could just as well say `osquery_helper_classes.ts` or `osquery_helper_functions.ts` :) )

You can call this `osquery_model.ts`, for example - here "model" stands for "data model". What do you think?